pkg/informer/informer.go

```diff
@@ -787,6 +890,96 @@ func newRESTMapper(fn func() (meta.RESTMapper, error)) restMapper {
 	}
 }
 
+type genericInformerBase[InformerType sharedIndexInformerBase, ListerType genericListerBase] interface {
+	Informer() InformerType
+	Lister() ListerType
+}
+
+type sharedIndexInformerBase interface {
```

Comment:
Isn't there already an interface upstream for this?

Reply:
Possibly, but I don't know it. Any pointer ?

Reply:
Seems to be part of `client-go` `tools/cache/SharedInformer` interface, but the `SharedInterface` contains more than that.
I haven't found any interface that only provides one or several of the methods we need here.

Reply:
I finally used `tools/cache/SharedIndexInformer`